client/src/pages/profile/index.tsx

```diff
@@ -335,86 +288,59 @@ export class ProfilePage extends React.Component<Props, State> {
   }
 
   render() {
-    const {
-      profile,
-      isEditingModeEnabled,
-      initialPermissionsSettings,
-      isInitialPermissionsSettingsChanged,
-      isInitialProfileSettingsChanged,
-      isProfileOwner,
-      connections,
-    } = this.state;
-
-    const isEditingModeVisible = initialPermissionsSettings && isEditingModeEnabled ? isEditingModeEnabled : false;
-    const isSaveButtonVisible = isInitialPermissionsSettingsChanged || isInitialProfileSettingsChanged;
+    const { profile, isProfileOwner, connections } = this.state;
+
+    const isEditingModeVisible = isProfileOwner;
```

Comment:
Why not just use `isProfileOwner`?

Reply:
Sure, piece by piece it eventually got shrinked to only `isProfileOwner`